websites/documentation/src/content/writing/release-notes.mdx

```diff
@@ -65,7 +65,13 @@ If there is no more, remove the marker above, too.
 - `title` (string, required): The title for the website
 - `description` (string, required): Max 256 characters plain text containing the change and its advantage ("tweet-able" and for RSS).
   256 = 280 twitter minus 23 for a link. Do not imply the title, this text is used stand-alone in Merchant Center and other feed recipients. It must convey the value proposition and key change topic.
-- `type` (string, optional): Must be either `feature` (for "New Feature"), `fix` (for "Resolved Issue") or `enhancement` (for "Enhancement")
+- `type` (string | string[], optional): Must be either `feature` (for "New Feature"), `fix` (for "Resolved Issue") or `enhancement` (for "Enhancement")
 - `topics` (array of string, optional): YAML array of free to choose text entries. The Filter UI may be constrained to pre-defined topics.
 - `published` (boolean, optional, defaults to `true`): `false` prevents rendering of the release in the site. **Should not be used in the main documentation**, reserved for atypical situations. Not included in the template.
 - `slug` (string, optional, defaults to `$date-$title` (slugified)): optional, allows overriding the autogenerated URL slug. **Should not be used in the main documentation**, reserved for migrating legacy docs.
+- `product` (string[], optional): optional, used in the release notes search to filter by product. If set, it overrides the `product` defined in the website `gatsby-configure.mjs`. Please note that in case an array is defined, just the first value is used for filtering purposes.
+- `productArea`: Option, used in the release notes search to filter by product area. If set, it overrides the `title` defined in the website `gatsby-configure.mjs`.
```

Comment:
@timonrey lets adjust the docs accordingly to the latest changes .. and remove the console logs that were missed